src/modules/ChannelList/dux/reducers.ts

```diff
@@ -39,15 +41,17 @@ export default function channelListReducer(
         return {
           ...state,
           loading: false,
+          silentLoading: false,
           error: null,
           allChannels: channelList,
           currentChannel,
         };
       })
-      .with({ type: channelListActions.FETCH_CHANNELS_START }, (action) => {
+      .with({ type: channelListActions.FETCH_CHANNELS_START }, () => {
         return {
           ...state,
           loading: true,
+          silentLoading: false,
```

Comment:
If we had a previous `allChannels`, but we're still fetching channels, wouldn't it trigger this `FETCH_CHANNELS_START` flow, which would reset `silentLoading` to false? Or is this flow something else?

Reply:
FETCH_CHANNELS_START is only ever called when 

```
const fetchChannelList = useFetchChannelList(...)
```

is called. This is included as part of the context, and as far as I can see it's never called.

```
      const {
        allChannels,
        fetchChannelList,
      } = useChannelListContext();
```